speech-commands/training/browser-fft/training_custom_audio_model_in_python.ipynb

```diff
@@ -746,11 +746,11 @@
         "tfjs.converters.save_keras_model(model, tfjs_model_dir)\n",
```

Comment:
<p>The path seems to be "/tmp/tfjs-model" but "/tmp/tfjs_model" had been used. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tfjs-models/pull/873/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>